backend/ipmi.pm

```diff
@@ -40,18 +40,28 @@ sub ipmi_cmdline {
 }
 
 sub ipmitool {
-    my ($self, $cmd) = @_;
+    my ($self, $cmd, $tries) = @_;
+    $tries //= 1;
```

Comment:
As the meaning of the third argument is not quite clear when it is just a number I suggest to pass a hash value instead, e.g. do:

```perl
my ($self, $cmd, %args) = @_;
$args{tries} //= 1;
...
for (1 .. $args{tries}) { ...
```

and call like

```perl
my $stdout = $self->ipmitool('chassis power status', tries => 3);
```

Reply:
Very good point :+1: